rgw/v2/tests/s3_swift/multisite_configs/test_bucket_sync_cmd_crash.yaml

```diff
@@ -2,7 +2,7 @@
 # script: test_Mbuckets_with_Nobjects.py
 config:
   user_count: 1
-  bucket_count: 2
+  bucket_count: 1
```

Comment:
Rather than changing the count of bucket, can we remove the hard coded name 
ex: https://github.com/red-hat-storage/ceph-qe-scripts/blob/test_rgw/rgw/v2/tests/s3_swift/test_Mbuckets_with_Nobjects.py#L214
https://github.com/red-hat-storage/ceph-qe-scripts/blob/test_rgw/rgw/v2/tests/s3_swift/test_Mbuckets_with_Nobjects.py#L428

Reply:
@anrao19 Thanks for the input, will work on it

Reply:
@anrao19 script will fail on secondary since we are assigning `bucket_name_to_create` randomly.
Currently we dont have that compatibility in ceph-qe-script to get correct bucket name while running it on secondary

Reply:
Sure @udaysk23